File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.py
class Solution(object):
    def intersection(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        arr = []
        i = 0
        while i<len(nums2):
            if nums2[i] in nums1 and  nums2[i] not in arr:
                arr.append(nums2[i])
            i+=1
        return arr

File: 0349-intersection-of-two-arrays/0349-intersection-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
var intersection = function(nums1, nums2) {
    let arr = [];
    let i =0
    while(i<nums2.length){
        if(nums1.includes(nums2[i])&&!arr.includes(nums2[i])){
            arr.push(nums2[i])
        }
        i++
    }
    return arr
};